CHANGELOG.MD

```diff
@@ -8,5 +8,5 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Released
 
-## [v0.1.0] - Tue Jun 11 2019
```

Comment:
Should the initial version be `0.1.0` or `1.0.0`?

Reply:
we spoke over it considering soved.,